src/conf/settings.xml

```diff
@@ -3160,6 +3160,8 @@ p, li { white-space: pre-wrap; }
             <valInt>0</valInt>
             <enumNames>GRB</enumNames>
             <enumNames>RGB</enumNames>
+            <enumNames>WRGB</enumNames>
+            <enumNames>RGBW</enumNames>
```

Comment:
This now duplicates the RGB vs. RGBW values together with the two values in the LED types. Could you instead change the list to the following:
- GRB(W)
- RGB(W)
- (W)RGB

And keep the white presence in the logic based on the LED type value? I know it makes the logic more complicated, but it keeps the options not being redundant, your change makes them redundant and that'd be confusing for users.